css-grid-1/reference/grid-layout-auto-columns-rows-reference.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
```

Comment:
Regarding the reference file, the ideal would be to use generic names describing the file.
Probably it's not possible in all the cases, but for example we've things like "grid-2x2-blue-yellow-lime-magenta.xht".
Also it'd be really great to reuse some of the reference files already existing. Maybe for a new test.

I cannot think a good name in this case, so probably the best approach is using the very same name than the test itself with "-ref" suffix.